Squest/settings.py

```diff
@@ -135,14 +137,14 @@
         'HOST': DB_HOST,
         'PORT': DB_PORT,
     },
-    'psql': {
-        'ENGINE': "django.db.backends.postgresql",
-        'NAME': DB_DATABASE,
-        'USER': DB_USER,
-        'PASSWORD': DB_PASSWORD,
-        'HOST': DB_HOST,
-        'PORT': DB_PORT,
-    }
+    # 'psql': {
+    #     'ENGINE': "django.db.backends.postgresql",
+    #     'NAME': DB_DATABASE,
+    #     'USER': DB_USER,
+    #     'PASSWORD': DB_PASSWORD,
+    #     'HOST': DB_HOST,
+    #     'PORT': DB_PORT,
+    # }
```

Comment:
Our current process doesn't propose to update settings directly. We use env variables. So this should be wraped by a condition that check if we are using postgress.